topology/Massachusetts Green High Performance Computing Center/PRP/MGHPCCCachingInfrastructure.yaml

```
@@ -0,0 +1,23 @@
+Production: true
+SupportCenter: Community Support Center
+GroupDescription: Caching infrastructure in Nebraska
+GroupID:
+
+Resources:
+ MGHPCC_NRP_OSDF_CACHE:
```

```suggestion
  MGHPCC_NRP_OSDF_CACHE:
```

minor whitespace change